timelock-agent/src/test/java/com/palantir/timelock/paxos/PaxosRemotingUtilsTest.java

```diff
@@ -141,14 +136,14 @@ public void addProtocolAddsHttpsIfSslPresent() {
 
     @Test
     public void addProtocolsAddsHttpIfSslNotPresent() {
-        assertThat(PaxosRemotingUtils.addProtocols(NO_SSL_TIMELOCK, ImmutableSet.of("foo:1", "bar:2")))
-                .containsExactlyInAnyOrder("http://foo:1", "http://bar:2");
+        assertThat(PaxosRemotingUtils.addProtocols(NO_SSL_TIMELOCK, ImmutableList.of("foo:1", "bar:2")))
+                .isEqualTo(ImmutableList.of("http://foo:1", "http://bar:2"));
     }
 
     @Test
     public void addProtocolsAddsHttpsIfSslPresent() {
-        assertThat(PaxosRemotingUtils.addProtocols(SSL_TIMELOCK, ImmutableSet.of("foo:1", "bar:2")))
-                .containsExactlyInAnyOrder("https://foo:1", "https://bar:2");
+        assertThat(PaxosRemotingUtils.addProtocols(SSL_TIMELOCK, ImmutableList.of("foo:1", "bar:2")))
+                .isEqualTo(ImmutableList.of("https://foo:1", "https://bar:2"));
```

Comment:
I would have thought `containsExactly` (without the `inAnyOrder` bit) would have been enough?